contrib/cirrus/runner.sh

```diff
@@ -176,66 +262,10 @@ msg "************************************************************"
 
 cd "${GOSRC}/"
 
-case "$TEST_FLAVOR" in
-    ext_svc) $SCRIPT_BASE/ext_svc_check.sh ;;
-    smoke)
-        make gofmt
-        # There is little value to validating commits after tag-push
-        # and it's very difficult to automatically determine a starting commit.
-        # $CIRRUS_TAG is only non-empty when executing due to a tag-push
-        # shellcheck disable=SC2154
-        if [[ -z "$CIRRUS_TAG" ]]; then
-            make .gitvalidation
-        fi
-        ;;
-    automation)
-        $SCRIPT_BASE/cirrus_yaml_test.py
-        req_env_vars CI DEST_BRANCH IMAGE_SUFFIX TEST_FLAVOR TEST_ENVIRON \
-                     PODBIN_NAME PRIV_NAME DISTRO_NV CONTAINER USER HOME \
-                     UID GID AUTOMATION_LIB_PATH SCRIPT_BASE OS_RELEASE_ID \
-                     OS_RELEASE_VER CG_FS_TYPE
-        bigto ooe.sh dnf install -y ShellCheck  # small/quick addition
-        $SCRIPT_BASE/shellcheck.sh
-        ;;
-    altbuild) altbuild ;;
-    build)
-        make podman-release
-        make podman-remote-linux-release
-        ;;
-    validate)
-        # Confirm compiile via prior task + cache
-        bin/podman --version
-        bin/podman-remote --version
-        make validate  # Some items require a build
-        ;;
-    bindings)
-        # shellcheck disable=SC2155
-        export PATH=$PATH:$GOSRC/hack
-        # Subshell needed for .cirrus.yml to find logformatter output in cwd
-        (cd pkg/bindings/test && ginkgo -trace -noColor -debug  -r) |& logformatter
-        ;;
-    endpoint)
-        make test-binaries
-        make endpoint
-        ;;
-    swagger)
-        build_swagger
-        # Cirrus-CI Artifact instruction expects file here
-        cp -v $GOSRC/pkg/api/swagger.yaml $GOSRC/
-        ;;
-    vendor)
-        make vendor
-        ./hack/tree_status.sh
-        ;;
-    docker-py) msg "This is docker-py stub, it is only a stub" ;;
-    unit) make localunit ;;
-    int) dotest integration ;;
-    sys) dotest system ;;
-    release)
-        if bin/podman info |& grep -Eq -- '-dev'; then
-            die "Releases must never contain '-dev' in output of 'podman info'"
-        fi
-        ;;
-    *)
-        die "Unknown/Unsupported \$TEST_FLAVOR=$TEST_FLAVOR" ;;
-esac
+handler="_run_${TEST_FLAVOR}"
```

Comment:
THANK YOU...this is ***SO*** much easier to read.

Reply:
Thank you!